hokusai/production.yml

```diff
@@ -103,3 +104,18 @@ spec:
     component: web
   sessionAffinity: None
   type: LoadBalancer
+  loadBalancerSourceRanges:
```

Comment:
Just added a new hokusai template config var `cloudflareIpSourceRanges` - if you want to use it this should work
```
  loadBalancerSourceRanges:
{%- for cidr_block in cloudflareIpSourceRanges %}
    - {{ cidr_block }}
{%- endfor %}
```

Reply:
cool.